src/pages/Products/PC/PCs.tsx

```diff
@@ -53,18 +52,32 @@ export const PCs = () => {
         "https://github.com/pawelNu/compstore-ui/assets/93542936/8196ca80-ef1b-4b67-a7bd-b56c7b7f23e3";
 
     const getPCs = useCallback(async () => {
-        await getPcsHandler(
-            filter,
-            setPCs,
-            setPagesCount,
-            setPageNumber,
-            setPageSize,
-        );
+        try {
+            const result = await axios.post(endpoints.pcs.getAll, filter);
+            setPCs(result.data.pcs);
+            const pagingMetadata: {
+                pagesCount: number;
+                pageNumber: number;
+                pageSize: number;
+            } = result.data.pagingAndSortingMetadata;
+            setPagesCount(pagingMetadata.pagesCount);
+            setPageNumber(pagingMetadata.pageNumber);
+            setPageSize(pagingMetadata.pageSize);
+            setLoading(false);
```

Comment:
Jak zakomentujesz `setLoading(false);` to spinner będzie się kręcił w nieskończoność.